pywinauto/controls/win32_controls.py

```diff
@@ -816,9 +816,9 @@ def set_edit_text(self, text, pos_start = None, pos_end = None):
                 aligned_text = six.binary_type(text)
 
         if isinstance(aligned_text, six.text_type):
-            buffer = ctypes.create_unicode_buffer(aligned_text, size=len(aligned_text) + 1)
+            buffer = ctypes.create_unicode_buffer(aligned_text, size=len(aligned_text.encode('unicode_escape')) + 1)
         else:
-            buffer = ctypes.create_string_buffer(aligned_text, size=len(aligned_text) + 1)
+            buffer = ctypes.create_string_buffer(aligned_text, size=len(aligned_text.encode('unicode_escape')) + 1)

```

Comment:
@SaadeMotion , you might noticed a failure in pywinauto.unittests.test_hwndwrapper.NotepadRegressionTests.testMenuSelectNotepad_bug 
```
Error message:
'ascii' codec can't decode byte 0xef in position 33: ordinal not in range(128)
Error stack trace:
Traceback (most recent call last):
  File "c:\python27\lib\unittest\case.py", line 329, in run
    testMethod()
  File "C:\projects\pywinauto\pywinauto\unittests\test_hwndwrapper.py", line 837, in testMenuSelectNotepad_bug
    self.app2.UntitledNotepad.Edit.set_edit_text(text)
  File "C:\projects\pywinauto\pywinauto\controls\win32_controls.py", line 821, in set_edit_text
    buffer = ctypes.create_string_buffer(aligned_text, size=len(aligned_text.encode('unicode_escape')) + 1)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xef in position 33: ordinal not in range(128)
```
It seems that the fix should be applied only for a unicode buffer, but not for simple strings